main.go

```diff
@@ -43,10 +43,11 @@ import (
 var (
 	app = kingpin.New("certigo", "A command line certificate examination utility.")
 
-	dump      = app.Command("dump", "Display information about a certificate from a file/stdin.")
-	dumpFiles = dump.Arg("file", "Certificate file to dump (or stdin if not specified).").ExistingFiles()
-	dumpType  = dump.Flag("format", "Format of given input (PEM, DER, JCEKS, PKCS12; heuristic if missing).").String()
-	dumpPem   = dump.Flag("pem", "Write output as PEM blocks instead of human-readable format.").Bool()
+	dump         = app.Command("dump", "Display information about a certificate from a file/stdin.")
+	dumpFiles    = dump.Arg("file", "Certificate file to dump (or stdin if not specified).").ExistingFiles()
+	dumpType     = dump.Flag("format", "Format of given input (PEM, DER, JCEKS, PKCS12; heuristic if missing).").String()
+	dumpPem      = dump.Flag("pem", "Write output as PEM blocks instead of human-readable format.").Bool()
+	dumpPassword = dump.Flag("password", "Password for PKCS12/JCEKS key stores (if required).").String()
```

Comment:
hm. maybe we should read it from a file instead? I'm not a fan of having a password flag/arg but somehow we need to automate it for tests.